youtube_dl/extractor/instagram.py

```diff
@@ -359,3 +360,132 @@ def _real_extract(self, url):
 
         return self.playlist_result(
             self._entries(data), username, username)
+
+
+class InstagramStoriesIE(InfoExtractor):
+    _VALID_URL = r'(?P<url>https?://(?:www\.)?instagram\.com/stories/(?:highlights/)?(?P<id>[^/?#&]+))'
+    _TESTS = [{
+        'url': 'https://www.instagram.com/stories/highlights/17870001829248418/',
+        'md5': '9e6417cf0e37addd3c9b5e96f71a6360',
+        'info_dict': {
+            'id': '1819943712034319556',
+            'title': 'Glow up🐯',
+            'ext': 'mp4',
+            'timestamp': 1531174217,
+            'uploader': 'lasvegasbarbie',
+            'upload_date': '20180709',
+        }
+    }, {
+        'url': 'https://www.instagram.com/stories/charmedplayhouses/',
+        'md5': '2e509994273520b99956ab57009d1b4f',
+        'info_dict': {
+            'id': '1864314268272813842',
+            'title': 'Stories - charmedplayhouses',
+            'ext': 'mp4',
+            'timestamp': 1536463597,
+            'uploader': 'charmedplayhouses',
+            'upload_date': '20180909',
+        },
+    }]
+
+    def _entries(self, reels_media, username, title=None):
+        entries = []
+        for obj in reels_media:
+            for item in obj.get('items', []):
+                if not item.get('is_video', False):
+                    continue
+                thumbnails = []
+                for thumb in item['display_resources']:
+                    thumbnails.append(dict(url=thumb['src'],
+                                           width=thumb['config_width'],
+                                           height=thumb['config_height']))
+                formats = []
+                if not title:
+                    title = 'Stories - %s' % username
+
+                for res in item['video_resources']:
+                    formats.append(dict(
+                        url=res['src'],
+                        format_id=res['profile'],
+                        width=res['config_width'],
+                        height=res['config_height'],
+                        format_note=res['mime_type'],
+                    ))
+
+                entries.append(dict(thumbnails=thumbnails,
+                                    thumbnail=item['display_url'],
+                                    id=item['id'],
+                                    title=title,
+                                    uploader=item['owner']['username'],
```

Comment:
Read coding conventions on optional meta fields. Fix all such meta fields added in this PR.